script/using_JuMP/working_with_data_files.jl

```diff
@@ -44,7 +44,9 @@ using XLSX
 
 #+
 
-excel_df = DataFrame(XLSX.readtable("data/SalesData.xlsx", "SalesOrders")...)
+current_dir = @__DIR__

```

Comment:
What is this used for?

Reply:
so that the script works if you call it from another folder, otherwise in the current state it expects you to be running this exactly at the script's location

Reply:
I meant, it doesn't appear to be used in the code?

Reply:
Indeed no, only data_dir is used

Reply:
fixed in last commit